HWKMETRICS-165

mwringe opened the pull request:
Add docker-push profile to push images out to a docker registry.

tsegismont commented on containers/pom.xml:
```diff
@@ -42,10 +42,32 @@
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jolokia</groupId>
```
This should be in the pluginsManagement/plugins section

tsegismont commented on containers/hawkular-metrics-docker/pom.xml:
```diff
@@ -95,6 +95,27 @@
+          <plugin>
+            <groupId>org.jolokia</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
+            <version>${docker-maven-plugin.version}</version>
```
You don't need to declare the version if the plugin is declared in the pluginsManagement/plugins section

tsegismont commented on containers/hawkular-metrics-docker/pom.xml:
```diff
@@ -81,7 +81,7 @@
             <groupId>org.jolokia</groupId>
             <artifactId>docker-maven-plugin</artifactId>
-            <version>0.11.2</version>
+            <version>${docker-maven-plugin.version}</version>
```
Same here

tsegismont commented on containers/hawkular-metrics-docker/pom.xml:
```diff
@@ -55,14 +55,14 @@
         <groupId>org.jolokia</groupId>
         <artifactId>docker-maven-plugin</artifactId>
-        <version>0.12.0</version>
+        <version>${docker-maven-plugin.version}</version>
```
Same here

tsegismont commented on containers/pom.xml:
```diff
@@ -42,10 +42,32 @@
+        <version>0.12.0</version>
+      </dependency>
+      <dependency>
+        <groupId>io.fabric8</groupId>
```
This should be in the pluginsManagement/plugins section

tsegismont commented on containers/cassandra-docker/pom.xml:
```diff
@@ -90,6 +90,27 @@
+          <plugin>
+            <groupId>org.jolokia</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
+            <version>${docker-maven-plugin.version}</version>
```
Same here

tsegismont commented on containers/cassandra-docker/pom.xml:
```diff
@@ -76,7 +76,7 @@
             <groupId>org.jolokia</groupId>
             <artifactId>docker-maven-plugin</artifactId>
-            <version>0.11.2</version>
+            <version>${docker-maven-plugin.version}</version>
```
Same here

tsegismont commented on containers/cassandra-docker/pom.xml:
```diff
@@ -50,14 +50,14 @@
         <groupId>org.jolokia</groupId>
         <artifactId>docker-maven-plugin</artifactId>
-        <version>0.12.0</version>
+        <version>${docker-maven-plugin.version}</version>
```
Same here

tsegismont commented on containers/Readme.md:
```diff
@@ -79,3 +79,15 @@ export KUBERNETES_CLIENT_CERTIFICATE_FILE=$OPENSHIFT_HOME/openshift.local.config
 export KUBERNETES_CLIENT_KEY_FILE=$OPENSHIFT_HOME/openshift.local.config/master/admin.key
 mvn clean install -Pcontainer-tests
 
+Deploying the Docker Images to a Registry
```
I've been able to run the container tests successfully this morning. So I've updated the doc. Also I've converted it to Asciidoc which is the standard on all H* projects. After you've fixed the pluginMgmt section issues I'll merge this PR manually and will convert this last paragraph to Asciidoc.

mwringe commented:
Updated to include requested fixes, updated to latest in master and squashed the commits together. Let me know if there are any more changes required.

tsegismont commented:
Looks good to me. Thanks for the update @mwringe . I'll merge as soon as Travis is happy (it failed due to an OOM error, I've just restarted the build)
